HealthVault/Classes/Models/MHVDateTimeBase.h

```diff
@@ -0,0 +1,30 @@
+//
+// MHVDateTimeBase.h
+// MHVLib
+//
+// Copyright (c) 2017 Microsoft Corporation. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#import <Foundation/Foundation.h>
+#import "MHVModelBase.h"
+
+@interface MHVDateTimeBase : MHVModelBase
+
+@property (nonatomic, strong) NSDate* date;
+
+- (NSString*)getDateFormatString;
```

Comment:
In objective-c we don't use a lot of 'get' prefixes. There are a few methods where we use this to have consistent naming with the OneSDK, if this is one of them then we should leave it. Otherwise, 'dateFormatString' would be more in line with the language.